Source/santasyncservice/SNTSyncPreflight.mm

```diff
@@ -281,6 +285,32 @@ BOOL Preflight(SNTSyncPreflight *self, google::protobuf::Arena *arena,
     }
   }
 
+  if constexpr (IsV2) {
```

Comment:
This method is very long. I'm OK if it's a TODO for now, but we should break it up a bit.

It might also be nice to unify the multiple "if V2" sections.